pkg/mkimage-raw-efi/install

```diff
@@ -492,6 +492,7 @@ dmidecode > "$REPORT/hardwaremodel.txt"
 # try to generate model json file
 ctr_run /opt/debug spec.sh > "$REPORT/controller-model.json"
 ctr_run /opt/debug spec.sh -v > "$REPORT/controller-model-verbose.json"
+ctr_run /opt/debug spec.sh -u > "$REPORT/controller-model-usb.json"
```

Comment:
Does it make sense to continue supporting device models assigning USB devices by PCI controllers?